tests/integration_tests/fixtures/registry.json

```diff
@@ -44,18 +44,14 @@
     },
     {
       "sourceDefinitionId": "9f32dab3-77cb-45a1-9d33-347aa5fbe333",
-      "name": "Non-published source",
-      "dockerRepository": "airbyte/source-non-published",
+      "name": "Docker-only source",
```

Comment:
@coderabbitai - This fixes a previous copy-paste error. Since this is a test fixture, there's no impact on core codepaths - only to fix the bug in the fixture.